pkg/kubernetes/scripts/graphs.js

```diff
@@ -541,8 +541,11 @@
                                 }
                             }
 
-                            var event = document.createEvent("CustomEvent");
-                            event.initCustomEvent("changed", false, false, null);
+                            var event = new CustomEvent("changed", {
+                                "bubbles":false,
+                                "cancelable":false,
+                                "detail":null
```

Comment:
Note that the keys don't need to be quoted (and we usually don't), and there's a space missing after the `:`.

Reply:
Thanks for pointing this out. Updated and (hopefully) remembered :)